User/UserRepository.php

```diff
@@ -20,4 +20,38 @@ public function findByUsername($username);
      * @return bool
      */
     public function supportsClass($class);
+
+    /**
+     * Add User to the repository.
+     *
+     * @param User $user
+     * @param bool $save
+     * @return void
+     */
+    public function add(User $user, $save = true);
+
+    /**
+     * Flush the repository.
+     *
+     * @param User|null $user
+     * @return void
+     */
+    public function save(User $user = null);
```

Comment:
Als je met doctrine werkt kan je meerdere bewerkingen opslaan tegelijk, het is mss zelf beter om geen parameters mee te geven